tachyon/c/zk/plonk/halo2/bn254_shplonk_prover_unittest.cc

```diff
@@ -127,7 +127,9 @@ TEST_P(SHPlonkProverTest, SetTranscript) {
           reinterpret_cast<PoseidonWriter<math::bn254::G1AffinePoint>*>(
               transcript->extra);
       digest_len = poseidon->GetDigestLen();
-      state_len = poseidon->GetStateLen();
+      // NOTE(chokobole): In case of Poseidon transcript,
+      // |tachyon_halo2_bn254_transcript_writer_update()| touches an internal
+      // member |absorbing_|. So |state_len| has to be updated after this call.
```

Comment:
Change `...|absorbing_|. So...` to one of:
1.  `...|absorbing_|, so...`
2.  `...|absorbing_|. Therefore, ...`
3. `...|absorbing_|; therefore, ...`
Just like "But," try not to use "So" at the beginning of a sentence.